schemas/context/application.schema.json

```diff
@@ -0,0 +1,70 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/application",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Application",
+  "type": "object",
+  "description":
+    "The entites related to Application usage.",
+  "definitions": {
+    "application": {
+      "properties": {
+        "@id": {
```

Comment:
Did the app scheme not exist at all?  Is is this in the PR because of some other change?  Are you reorganizing application along with moving the metrics around?

Reply:
It was already in the schema, but was strangely referenced through channels. I refactored t to be consistent with the other structures (commerce etc). I also then had to clean up the \channels\ in how it referred to applications. So really just refactoring, nothing net new.